tests/test9.1.yaml

```diff
@@ -0,0 +1,30 @@
+# Tests for rtl87xx boards using LibreTiny
+---
+wifi:
+  ssid: "ssid"
+
+rtl87xx:
+  board: generic-rtl8710bn-2mb-788k
+  framework:
+    version: dev
```

Comment:
Why version dev? When merged, we're not going to default to `dev` but `recommended`. The framework block can be removed.